src/api/db/migrate/001_initial_database.rb

```diff
@@ -915,6 +855,79 @@ def self.up
707061034),(20100805100000),(20100812100000),(20100827100000),(20100903100000),(20100927110821),(20100927132716),(20100927133955),(20100928081344),(20101110100000),(20110117000000),(20110131100000),(20110202100000),(20110202110000),(20110302100000),(20110303100000),(20110309100000),(20110318112742),(20110321000000),(20110322000000),(20110323000000),(2011033000000),(20110331001200),(20110404085232),(20110404085325),(20110404090700),(20110405151201),(20110502100000),(20110519000000),(20110527000000),(20110615083665),(20110615083666),(20110627001200),(20110703001200),(20110711001200),(20110719142500),(20110725105426),(20110728072502),(20111005000000),(20111116100002),(20111117162400),(20111122000000),(20111123000000),(20111206000000),(20111206151500),(20111207000000),(20111213000000),(20111215094300),(20111303000000),(20120110094300),(20120110104300),(20120111094300),(20120112094300),(20120112194300),(20120119194300),(20120119204300),(20120119204301),(20120120104301),(20120120114301),(20120124114301),(20120124114302),(20120124114303),(20120216114303),(20120217114303),(20120217114304),(20120220114304),(20120222105426),(20120223105426),(20120304205014),(20120312204300),(20120313113554),(20120313131909),(20120319104301),(20120319133739),(20120320134850),(20120407173644),(20120411112931),(20120411121152),(20120417115800),(20120418121859),(20120529150500),(20120903122955),(20120904122955),(20120907114304),(20121014124846),(20121015121807),(20121112104534),(20121112110642),(20121114093616),(20121120110642),(20121120124300),(20121121142111),(20121124032111),(20121130103300),(20121130143300),(20121213140751),(20121213144129),(20121216151549),(20121220151549),(20130111085930),(20130220160000),(20130301100000),(20130409123324),(20130410124738),(20130414061002),(20130603100244),(20130610100244),(20130612151549),(20130618083665),(20130621083665),(20130626160000),(20130627193722),(20130702083665),(20130702203665),(20130723055536),(20130725123636),(20130726144516),(20130802183104),(20130802183717),(20130802190951),(20130805073101),(20130807071147),(20130814071147),(20130816183104),(20130817082602),(20130819114303),(20130820151442),(20130830043205),(20130903114302),(20130904071147),(20130910162318),(20130917124132),(20130920090004),(20130930130128),(20131005225515),(20131006000000),(20131006162847),(20131020151037),(20131020165316),(20131021063641),(20131022114302),(20131023063641),(20131027122410),(20131028085325),(20131029112259),(20131105112259),(20131106112233),(20131111090310),(20131111193512),(20131111194720),(20131112140033),(20131120193512),(20131123113417),(20131124071042),(20131125071042),(20131125101042),(20131126074753),(20131126205430),(20131127091624),(20131209095749),(20131209103450),(20131210182719),(20131218071042),(20140113110551),(20140123063641),(20140123071042),(20140124071042),(21),(22),(23),(24),(25),(26),(27),(28),(29),(3),(30),(31),(32),(33),(34),(35),(36),(37),(38),(39),(4),(40),(41),(42),(43),(44),(45),(46),(47),(48),(5),(6),(7),(8),(9);
```

Comment:
(as discussed on phone), we can drop them, but we need to adapt the test to ignore the missing entries then.